redis/tests/test_types.rs

```diff
@@ -77,6 +81,10 @@ fn test_i32() {
 
         let bad_i: Result<i32, _> = parse_mode.parse_redis_value(Value::SimpleString("42x".into()));
```

Comment:
minor - can you move `Value::SimpleString("42x".into())` to a variable, just to be clear that both calls use the same value?

Reply:
Both of these values doesn't need to use the same value in this tests but refactor it anyway